docker-compose-native/docker-compose.yml

```diff
@@ -68,19 +66,20 @@ services:
         ipv4_address: 172.19.0.4
   worker_1:
     container_name: worker_1
-    image: public.ecr.aws/oss-presto/presto-native:0.286-20231205020058-ada5eb3-amd64
+    image: public.ecr.aws/oss-presto/presto-native:0.289-20240712181104-ba08036
```

Comment:
0.288 is the amd64 version. or use `latest`, too.

Reply:
0.289-20240712181104-ba08036 could be deleted later on, please use prestodb/presto:0.288 for the java cluster workers.

Reply:
Thanks for the suggestion, this file is for the native workers and v0.288 is working so updated accordingly. Could you PTAL?